uiowa/uiowa
# Find text paragraphs fixes

There's an issue where multiple paragraph hits in a single page will overwrite each other.




# How to test



- `git checkout find-text-p-fixes && git pull`
- `blt ds --site=sandbox.uiowa.edu`
- `drush @sandbox.local uli /admin/find-text`
- Search for 'accordion', or 'table', or something similar
- See that you get more than one hit for node 26 (https://sandbox.local.drupal.uiowa.edu/node/26/)
